src/TeamSyncer.ts

```diff
@@ -208,11 +210,22 @@ export class TeamSyncer {
         }
     }
 
-    public async syncUser(teamId: string, domain: string, item: ISlackUser) {
+    public async syncUser(teamId: string, domain: string, item: ISlackUser, newUser = false) {
         log.info(`Syncing user ${teamId} ${item.id}`);
         const slackGhost = await this.main.ghostStore.get(item.id, domain, teamId);
         if (item.deleted !== true) {
             await slackGhost.updateFromISlackUser(item);
+            for (const teamRoom of this.main.rooms.getBySlackTeamId(teamId)) {
+                if (teamRoom.IsPrivate || teamRoom.SlackType !== "channel") {
+                    // We only want PUBLIC rooms
+                    return;
```

Comment:
I think you meant to skip of this one room, but not terminate the sync for the user.
```suggestion
                    continue;
```